### DISCUSSION TITLE
FluentButton - Bold Text

### DISCUSSION THREAD

**Discussion by AlexHedley** [microsoft/fluentui-blazor#3075] [Q&A] [answered]
Is there a property you can set to make the button text **bold**?

Links

- https://www.fluentui-blazor.net/Button

**Comment by coderdnewbie** [comment, marked as answer]
Set the style to font-weight: bold; for example:

`
<FluentButton Appearance="Appearance.Accent" style="font-weight: bold;" @onclick=" IncrementCount">Click me</FluentButton>
`

**Reply by AlexHedley** [reply]
Thanks
